services/userService.js

```diff
@@ -0,0 +1,61 @@
+const User = require("../models/user");
+
+async function createUser(data) {
```

Comment:
The function declaration is incomplete. It should have a return statement or a thrown error. Consider adding a try-catch block to handle potential errors.